src/compose.rs

```diff
@@ -0,0 +1,8 @@
+//! Constructing octets sequences from data.
+//!
+//! Composing encoded data always happens directly into an octets builder.
+//! Therefore, no `Composer` type is necessary. This module only defines a
+//! trait [`Compose`] which is used as an extension trait to provide
+//! `compose` methods for built-in types.
+
+// XXX Add the Compose trait or remove the module.
```

Comment:
Want to make that decision, or just leave the XXX for now?

Reply:
I had already removed the module from `lib.rs`, so that decision’s been made …